data/disney

```diff
@@ -54,10 +54,6 @@ babyzone.com
 bamgrid.com
 bam.nr-data.net
 beautyandthebeastmusical.co.uk
-braze.com
-cdn.optimizely.com
-cdn.registerdisney.go.com
```

Comment:
Sorry, I chose the wrong line. `cdn.registerdisney.go.com` should not be removed.